recipes/sctk/0001-fix-compiler-for-rfilter1.patch

```diff
@@ -1,25 +0,0 @@
-From 0f0f42fa908ac3812cee91645102c440f8f542c7 Mon Sep 17 00:00:00 2001
-From: Unal Ege Gaznepoglu <uenal.ege.gaznepoglu@iis.fraunhofer.de>
```

Comment:
Do not delete other recipes from the recipes folder. It can cause merge conflicts.

Reply:
This was a recipe that I pushed some time ago. I did not know back then that I could not fork the same repository twice. How should I treat this, by editing the commit history?

Reply:
You can use `git rebase`. Multiple forks of the same repository are not necessary because you can have multiple branches in a fork.